tests/cube/api/editoverview.api.test.ts

```diff
@@ -193,6 +194,7 @@ describe('Edit overview API', () => {
     const owner = createUser({ id: '12345' });
     const existingCube = createCube({ shortId: 'foo', owner });
     const updatedCube = createCube({
+      ...existingCube,
```

Comment:
More accurate setup with the updated cube being the existing one but with specific things updated